.github/ISSUE_TEMPLATE/4-constraint-work.yaml

```
@@ -0,0 +1,112 @@
+name: Create or Revise a Constraint
+description: Define work to be performed specifically related to the creation, refinement, or decommissioning of constraints.
+labels: ["enhancement"]
+body:
+  - type: textarea
+    id: user-story
+    attributes:
+      label: Constraint Task
+      description: Identify the constraint 
+      placeholder: |
+        Consistent with the parent/tracked-by issue, this constraint work focuses on ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: intended-outcome
+    attributes:
+      label: Intended Outcome
+      description: Describe the reason for the constraint 
+      placeholder: |
+        Ensure the {*content*} is ...
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: OSCAL Content Details
+      
+  - type: dropdown
+    attributes:
+      label: Syntax Type
+      options:
+      - This is required core OSCAL syntax.
+      - This is optional core OSCAL syntax.
+      - This is a FedRAMP extension.
+      - This is a mix of required, optional, and/or extended syntax.
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Allowed Values
+      options:
+      - There are no relevant allowed values.
+      - There are only NIST-defined allowd values.
+      - FedRAMP allowed values must be defined or verified.
+      - NIST-allowed values must be extended with FedRAMP allowed values.
```

Do we think on initial submission for more junior staff and/or community contributors there should not be a "not sure"?

```suggestion
      - NIST-allowed values must be extended with FedRAMP allowed values.
      - Not sure, can maintainers help me choose?
```